config/device-classes/generic/clavister.yaml

```
@@ -0,0 +1,45 @@
+name: clavister
+
+config:
+  components:
+    cpu: true
+    memory: true
+
+match:
+  logical_operator: "OR"
+  conditions:
+    - type: SysObjectID
+      match_mode: startsWith
+      values:
+        - ".1.3.6.1.4.1.5089.1"
+
+identify:
+  properties:
+    vendor:
+      - detection: constant
+        value: "Clavister AB"
+    model:
+      - detection: SysDescription
+        operators:
+          - type: modify
+            modify_method: regexSubmatch
+            regex: '^([\D]+)'
```

Add Clavister as prefix so that Clavister isn't contained in the model description.
```suggestion
            regex: '^Clavister ([\D]+)'
```